src/hd.rs

```diff
@@ -375,33 +308,33 @@ fn batch_import(rpc: &RpcClient, import_reqs: Vec<(Address, RescanSince, String)
 
 #[derive(Debug, Clone, PartialEq)]
 pub enum KeyOrigin {
-    Derived(Fingerprint, u32),
+    Derived(DescriptorChecksum, u32),
     Standalone,
     // bwt never does hardended derivation itself, but can receive an hardend --bare-xpub
-    DerivedHard(Fingerprint, u32),
+    DerivedHard(DescriptorChecksum, u32),
```

Comment:
The `DerivedHard` variant is no longer necessary, the descriptor itself already includes this information.

Reply:
Fixed